src/app/cube/cube.routing.ts

```diff
@@ -34,6 +37,15 @@ const cube_routes: Routes = [
         path: 'about',
         component: AboutUsComponent,
       },
+      {
+        path: 'contribute-page',
```

Comment:
how do people feel about renaming this to just contribute instead of contribute-page?
```suggestion
        path: 'contribute',
```

Reply:
I'm fine with it as long as we have no other interfering name.m